.github/workflows/linux_job.yml

```diff
@@ -68,6 +68,13 @@ on:
         required: false
         default: ''
         type: string
+      pytest-results:
```

Comment:
Might also be good to just make this a direct directory that people can utilize like `${RUNNER_TEST_RESULTS}` that can be generic enough to handle basically anything we throw into it.

Reply:
Instead of giving people the option to choose where to store these we just have a dedicated place to make it easier.

Reply:
How would you design the interface in this case? Does the user still have to set a flag that they want to surface the failing tests or will that happen automagically if they the `${RUNNER_TEST_RESULTS}` directory has some files in them?

Reply:
I'd say the latter. In my mind the perfect interface would just give people the directory ("${RUNNER_TEST_RESULTS}") as a directory where you can put any test results that you want and the generic job should try to display it as best as it can.